airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/ConnectionTransformationTab/components/DbtCloudTransformationsCard.tsx

```diff
@@ -17,9 +17,9 @@ import { useCurrentWorkspace } from "hooks/services/useWorkspace";
 import { DbtCloudJob, isSameJob, useDbtIntegration, useAvailableDbtJobs } from "packages/cloud/services/dbtCloud";
 import { RoutePaths } from "pages/routePaths";
 
-import dbtLogo from "./dbt-bit_tm.svg";
+import dbtLogo from "../dbt-bit_tm.svg";
+import octaviaWorker from "../octavia-worker.png";
```

Comment:
Why all these import sorts? Is this from our new rule?

Reply:
I just changed the structure of ConnectionItemPage to make it more readable, but I will return to the previous implementation as discussed before